Sources/AsyncConnection.swift

```diff
@@ -3,7 +3,7 @@ public protocol AsyncConnection: AsyncStream {
 }
 
 extension AsyncConnection {
-    public func open(result: (Void throws -> AsyncConnection) -> Void) throws {
+    public func open(_ result: (Void throws -> AsyncConnection) -> Void) throws {
```

Comment:
@dreymonde what did you mean by this?
>especially these kinds of tupealiases

Personally I think that tupealiases are only ok in two places: where they seriously reduce boilerplate, or when they make things more clear. I think in this case it satisfies both constraints and it makes a lot of sense to use it.